src/OpenTelemetry/Activities.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.Extensions.Logging;
+using Temporalio.Activities;
+
+namespace TemporalioSamples.OpenTelemetry
```

Comment:
Use the top-of-file namespace declaration style like is done in the workflow class